independent-publisher-connectors/Decentraland/readme.md

```diff
@@ -0,0 +1,26 @@
+# Decentraland
+The Decentraland platform is a metaverse where people can create avatars, purchase virtual LAND parcels, buy and sell NFTs, and participate in the Decentraland DAO.  This connector allows a user to pull LAND parcel data into their app, flow, or report.
+
+## Publisher: Roy Paar
+
+## Prerequisites
+None.
+
+## Supported Operations
+### Get Districts
+Get a list of all districts within Decentraland.  Returns all parcels within each district with x, y coordinates.
+
```

Comment:
There is no Obtaining Credentials section. Please refer to this [link ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md)to see the official template which you should follow.

Reply:
Fixed